automatminer/automl/gen_evolve_v2.py

```diff
@@ -0,0 +1,101 @@
+import random
+
+class NeuralNetOptimizer():
+    def __init__(self, X, y, model, retain = 5, random_select=0.05, mutate_chance=0.05, count=15):
+        self.random_select = random_select
+        self.mutate_chance = mutate_chance
+        self.retain = retain
+        self.X = X
+        self.y = y
+        self.Model = model
+        ACTIVATION = ["sigmoid", "tanh", "relu", "elu"]
+        OPTIMIZERS = ["sgd", "rmsprop", "adagrad", "adadelta", "nadam", "adamax", "adam"]
+        self.NN_PARAM_CHOICES = {"units": range(1, 1000), "hidden_layer_sizes": range(1, 3), "optimizer": OPTIMIZERS,
+                            "activation": ACTIVATION}
+        pop = self.create_population(count)
+        for _ in range(10):
+            pop = self.evolve(pop, X, y)
+
+        self.model_win = self.best_model(pop)
+
+    def create_population(self, count):
```

Comment:
if possible, I'd try to use more descriptive variable names than `count`. For example, `pop_size`, which is more reflective of what the variable actually is: population size.